docs/web/generating-code.mdx

```diff
@@ -139,12 +144,18 @@ response classes. You can see them being imported for the service definition.
 To learn more about `protoc-gen-es`, head over to the documentation for the
 [Protobuf-ES project](https://github.com/bufbuild/protobuf-es/blob/main/docs/generated_code.md).
 
+If your bundler does not handle the `.js` extension in the import `from "./eliza_pb.js"`
+correctly, you can configure it following our examples [here](http://localhost:3000/docs/web/supported-browsers-and-frameworks#webpack),
+or you can add the plugin option `import_extension=none` to remove the extension.
+
+You can find the documentation for all available [plugin options on npmjs.com](https://www.npmjs.com/package/@connectrpc/protoc-gen-connect-es#plugin-options).
+
```

Comment:
This should hopefully help users to find the workaround for this common gotcha more quickly.